### ISSUE TITLE
[.NET 6] Blazor cannot find member named xxx while debugging

### ISSUE THREAD

**Issue by BearsPunch** [dotnet/runtime#60832]
### Describe the bug
Hi. I tried to upgrade my project to .net 6 rc and met problem when debugging my app.

If I set breakpoint inside my method (event method for dropdown element), Blazor throws exceptions and never step to this breakpoint again until reload project. 

Blazor send this:
`fail: Microsoft.WebAssembly.Diagnostics.DevToolsProxy[0] sending error response for id: msg-52D572BE3642D3EDFFB06D7556619E0D:::1028 -> [Result: IsOk: False, IsErr: True, Value: , Error: {   "result": {     "type": "object",     "subtype": "error",     "description": "Cannot find member named 'GetTranslationsFromServer'.",     "className": "ReferenceError"   } } ]
fail: Microsoft.WebAssembly.Diagnostics.DevToolsProxy[0] sending error response for id: msg-52D572BE3642D3EDFFB06D7556619E0D:::1031 -> [Result: IsOk: False, IsErr: True, Value: , Error: {   "result": {     "type": "object",     "subtype": "error",     "description": "Cannot find member named 'Enum'.",     "className": "ReferenceError"   } } ]
fail: Microsoft.WebAssembly.Diagnostics.DevToolsProxy[0] sending error response for id: msg-52D572BE3642D3EDFFB06D7556619E0D:::1040 -> [Result: IsOk: False, IsErr: True, Value: , Error: {   "result": {     "type": "object",     "subtype": "error",     "description": "Cannot find member named 'Enum'.",     "className": "ReferenceError"   } } ]`

My method just do this:
```
var language = LanguageHelper.GetSupportedCultureByLanguage(
                (SupportedLanguages)Enum.Parse(typeof(SupportedLanguages), selectedLanguage));
```

```
public string GetSupportedCultureByLanguage(SupportedLanguages language)
        {
            return language switch
            {
                SupportedLanguages.English => "en-GB",
                SupportedLanguages.Polish => "pl-PL",
                SupportedLanguages.Russian => "ru-RU",
                _ => "pl-PL",
            };
        }

```

### To Reproduce
Create any test enum, for example:
```
public enum TestEnum
    {
        Test,
        Test2
    }
```

after that try to debug this line of code
`Enum.Parse(typeof(TestEnum), "Test");`

For new project, I see exception but debugger continues to work. For my own project, as I said, debugger never hit breakpoint again

Browser console is always empty

### Further technical details
- NET 6.0.100-rc.2.21505.57
- Visual Studio 2022 Community RC2

**Comment by weirdyang** [comment]
II'm getting this issue with .net 5

![image](https://user-images.githubusercontent.com/25576022/154044375-920ba840-7789-4c75-80c8-c037ce4d7ae4.png)

**Comment by thaystg** [comment]
1) Redirect bug is fixed on dotnet 6.0.
2) This error is expected "Cannot find member named" is expected because VS tries to evaluate a lot of expressions that are not supported yet by the debugger.
